.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+| Question         | Answer
+| ---------------- | ---
+| Branch?          | `master` for hotfix / `develop` for bug fixes <!-- see below -->
+| Bug fix?         | yes/no
```

Comment:
```suggestion
| Bug fix?         | [ ]
```
?

Reply:
We cannot draw easily a checkbox.

See: https://stackoverflow.com/questions/47344571/how-to-draw-checkbox-or-tick-mark-in-github-markdown-table

Reply:
Well maybe we don't need a table then 😄

Reply:
A Todolist can be confusing because a Pull Request will never be seen as **completed**.
That's why using a table with questions like `yes/no` look more affordable.
Let me know what do you think.

Thanks!